beacon_node/rest_api/src/helpers.rs

```diff
@@ -31,6 +36,24 @@ pub fn parse_root(string: &str) -> Result<Hash256, ApiError> {
     }
 }
 
+/// Parse a PublicKey from a `0x` prefixed hex string
+pub fn parse_pubkey(string: &str) -> Result<PublicKey, ApiError> {
```

Comment:
I think this function permits both with and without the `0x` prefix. I would have thought the `0x` prefix is mandatory.

Reply:
Yeah it does; I actually have the if statement around it that checks this commented out.
I will uncomment it.